src/tests/ftest/osa/osa_offline_drain.py

```diff
@@ -114,7 +114,8 @@ def test_osa_offline_drain(self):
 
         Test Description: Validate Offline Drain
 
-        :avocado: tags=all,pr,hw,large,osa,osa_drain,offline_drain
+        Disabled from pr until DAOS-6107 is fixed.
+        :avocado: tags=all,hw,large,osa,osa_drain,offline_drain
```

Comment:
Would it be better to use the https://github.com/daos-stack/daos/blob/e8b0b93f0b7f49bc07ab30b681a06bb3e623337c/src/tests/ftest/util/apricot/apricot/test.py#L62 decorator to skip these tests so that they are at least reported in the skipped ticket list and not totally forgotten?

The other concern I always have with disabling tests such as this is that there is nothing in the process of fixing DAOS-6107 that **ensuress** this test gets re-enabled again.  One would hope that the process of fixing DAOS-6107 means that the person fixing it notices there is a test for it that has been disabled, but there is no assurance of that happening.

Reply:
I would be happy to use this. Would you have any example?
As for whether the test will be re-enabled, i will take care of this with the relevant engineers.

Reply:
Found an example actually ... going to update the patch.